applications/secux_hardware_wallet_support_polkadot_transactions_and_staking.md

```diff
@@ -0,0 +1,188 @@
+| **0c.** | Testing and Testing Guide               | We will supply a detailed testing guide along with a SecuX hardware wallet to thoroughly verify the new functionalities. This approach will ensure that all features are tested under real-world conditions, providing users with reliable and secure solutions within the ecosystem.                                                                                                                                         |
+|     0e. | Article                                 | We will create and publish an in-depth article that explores the various supports and functionalities of the Polkadot network. Additionally, we will enhance the article's reach through a co-marketing initiative on social media with Polkadot. This strategic partnership will not only educate our audience about Polkadot's capabilities but also amplify our content's visibility and engagement across platforms.      |
+|      1. | Periodically Meeting                    | Collaborative Efforts and Communication, Polkadot Integration of SecuX SDK support.                                                                                                                                                                                                                                                                                                                                           |
+|      2. | SecuX SDK Polkadot support npm version  | Development of SDK API for Polkadot.js.                                                                                                                                                                                                                                                                                                                                                                                       |
+|      3. | SecuX SE Firmware version number        | Development of Secure Element (SE) Firmware.                                                                                                                                                                                                                                                                                                                                                                                  |
+|      4. | SecuX MCU Firmware version number       | Development of MCU Firmware including transection confirmation, transection details, account balance.                                                                                                                                                                                                                                                                                                                         |
```

Comment:
Why are these called `version number`?

Reply:
We will have specific SE firmware version and MCU firmware version supporting Polkadot.

Reply:
I understand, I'm just a bit confused why that's the name of the deliverable when the actual work being done for it is software development. Could you expand a bit on what exactly needs to be implemented for these deliverables that's specific to Polkadot? Account generation?

Reply:
To implement and verify compatibility with BIP32-ED25519 seed phrase recovery, ensuring that accounts generated by Ledger are compatible with SecuX and vice versa. Users can switch between Ledger and SecuX devices without worrying about compatibility issues.

2. Display the Polkadot (DOT) cryptocurrency icon on the front end of the SecuX Web Wallet app.

3. Support Polkadot (DOT) cryptocurrency send/receive functions, including transaction history, via the SecuX Web Wallet and SecuX Hardware Wallet through WebUSB or Bluetooth.

4.Prompt the user to confirm the transaction amount and receiver address on the SecuX Hardware Wallet.